test/server/triggers.ts

```diff
@@ -0,0 +1,75 @@
+import { expect, test } from 'vitest'
+import { app } from './utils'
+
+test('trigger list filtering', async () => {
```

Comment:
```suggestion
test('trigger list with limit', async () => {
```

Reply:
also same as for other list tests, we should create some first to check that they are returned and that limit works